src/modules/modals/CreateRow.vue

```diff
@@ -85,25 +100,14 @@ export default {
 			return !!value || value === 0
 		},
 		async actionConfirm() {
-			let mandatoryFieldsEmpty = false
-			this.columns.forEach(col => {
-				if (col.mandatory) {
-					const validValue = this.isValueValidForColumn(this.row[col.id], col)
-					mandatoryFieldsEmpty = mandatoryFieldsEmpty || !validValue
-				}
-			})
-			if (!mandatoryFieldsEmpty) {
```

Comment:
Since we now ensure that the "Save" button is disabled if there are empty mandatory fields, I assume it's safe to remove this check